terraform/shared_account_pathtolive_infra_ci/promotion_pipeline_path_to_live.tf

```diff
@@ -135,7 +135,6 @@ resource "aws_codepipeline" "path_to_live" {
       provider        = "CodeBuild"
       version         = "1"
       namespace       = "ImageHashes"
-      run_order       = 1
```

Comment:
This needs to happen before the deploy as it overwrites tags, we may be able to drop it completely, but to be safe I'd set this and the restart job to run in parallel, then do the deploy job